src/evaluateUpdatingExpression.ts

```diff
@@ -18,10 +18,12 @@ import { Language, Logger } from './types/Options';
  * @public
  */
 export type UpdatingOptions = {
+	annotateAst?: boolean;
 	debug?: boolean;
 	disableCache?: boolean;
 	documentWriter?: IDocumentWriter;
 	logger?: Logger;
+	logUnannotatedQueries?: boolean;
```

Comment:
Do we expect external people to use this flag? I do not think it has much use. Or do you use it in the demo page?

Reply:
Anyway, a bit of docs would be awesome here

Reply:
See the docs at the other Options type. Ideally, this UpdatingOptions would be a type union between Options and the additional options, but a duplicate declaration is fine here.

Reply:
hmmm, good question. We kept everything optional to keep breakage to a minimum, but this option is most certainly primarily for debug purposes. It should be safely removed once the annotation itself is finalized.

For now I will document it better, including that it can be removed once the annotation is done.